subsys/zigbee/cli/zigbee_cli_utils.c

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <zephyr/types.h>
+#include <shell/shell.h>
+
+#include <zboss_api.h>
+#include <zb_error_handler.h>
+#include <zigbee_logger_eprxzcl.h>
+#include "zigbee_cli_utils.h"
+
+extern zb_uint8_t cli_agent_ep_handler_attr(zb_bufid_t bufid);
+extern zb_uint8_t cli_agent_ep_handler_generic_cmd(zb_bufid_t bufid);
+extern zb_uint8_t cli_agent_ep_handler_report(zb_bufid_t bufid);
+extern zb_uint8_t cli_agent_ep_handler_ping(zb_bufid_t bufid);
+
+zb_device_handler_t zb_ep_handlers[] = {
```

Comment:
The idea behind using named sections was to be able to select a subset of CLI commands that will be included.
I see that this implementation does not have this feature.
Do you think that the ability to get only a subset of CLI commands is worth implementing?

Reply:
It can be done but I'd go with KConfig options to set which subsystem to have included.
The only reason I can think of to have such feature is a memory limitation. Is this reason "good enough" to implement this?